examples/static-hbase/README.md

```diff
@@ -3,13 +3,16 @@ This example depends on an existing VPC and subnet with configurations that meet
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
 ## Requirements
 
-No requirements.
+| Name | Version |
+|------|---------|
+| aws | 3.27.0 |
```

Comment:
I'm curious as to what changed to make these requirements appear?

Reply:
oh, maybe it's because I've run the examples before and I have a .terraform folder here.